.circleci/config.yml

```diff
@@ -35,6 +35,9 @@ parameters:
 
 workflows:
   lint-pack:
+    when:
+      not:
+        equal: [ main, << pipeline.git.branch >> ]
```

Comment:
Doesn't this mean that this job will never run?

Reply:
Itll only be ran on personal branches, so this is the check before you can merge